finetune/input_pipeline.py

```diff
@@ -122,49 +123,29 @@ def _array_format(self, encoded_output):
             1: positional embedding
         """
         seq_length = len(encoded_output.token_ids)
-        x = np.zeros((seq_length, 2), dtype=np.int32)
+        x = np.zeros((seq_length), dtype=np.int32)
```

Comment:
we probably don't need to allocate this array of zeros anymore.  could just say x = `encoded_output.token_ids`